lib/zendesk_apps_tools/command.rb

```diff
@@ -42,18 +48,14 @@ def new
       @app_name     = get_value_from_stdin("Enter a name for this new app:\n",
                                            error_msg: invalid.call('app name'))
 
-      @iframe_location = if options[:v1]
-                           '_legacy'
-                         else
-                           iframe_uri_text = 'Enter your iFrame URI or leave it blank to use'\
-                                             " a default local template page:\n"
-                           get_value_from_stdin(iframe_uri_text, allow_empty: true, default: 'assets/iframe.html')
-                         end
+      @iframe_location = iframe_uri_text = 'Enter your iFrame URI or leave it blank to use'\
+                                           " a default local template page:\n"
+                         get_value_from_stdin(iframe_uri_text, allow_empty: true, default: 'assets/iframe.html')
 
       prompt_new_app_dir
 
-      skeleton = options[:v1] ? 'app_template' : 'app_template_iframe'
```

Comment:
We should remove the app_template folder and any v1 related tests that are no longer relevant.